lib/ansible/module_utils/eos.py

```diff
@@ -24,6 +25,7 @@
     port=dict(type='int'),
     username=dict(required=True),
     password=dict(no_log=True),
+    ssh_keyfile=dict(type='path'),
```

Comment:
need to add this argument to the doc fragment as well at utils/module_docs_fragments/eos.py